### DISCUSSION TITLE
Can't open UTM (installed via Trollstore)

### DISCUSSION THREAD

**Discussion by ChrisGleich1AufsMaul** [utmapp/UTM#4885] [UTM Help (iOS)]
**Describe the issue**
UTM stopped woking at some point and crashes now on startup (unjailbroken state). Jailbroken it launches, but says when trying to run a VM: "JIT buffer: invalid argument", although it's launched by JitStreamer.

**Configuration (required)**
* UTM Version: 4.1.4 (also tried older versions)
* OS Version: iOS 15.1
* Device Model: iPad Air 4th gen
* Is it jailbroken (name jailbreak used)?: Jailbroken with Xina15
* How did you install UTM? Trollstore

**Crash log**
Not available

**Debug log**
attached here
[debug.log](https://github.com/utmapp/UTM/files/10331996/debug.log)

![IMG_0114](https://user-images.githubusercontent.com/49039734/210241423-0b88c3d8-e2ca-4dfd-bb3f-d86582db70a2.png)

**Comment by osy** [comment]
You either tried to assign too much memory or too much JIT cache.

**Comment by ChrisGleich1AufsMaul** [comment]
Thanks, but that shouldn‘t be the problem 🤔
<img width="1180" alt="Bildschirmfoto 2023-01-04 um 13 24 13" src="https://user-images.githubusercontent.com/49039734/210554472-a11e254f-e250-4e5e-8d1b-a9fe4f0a23f9.png">
